oss_src/unity/python/sframe/cython/cy_flexible_type.pyx

```diff
@@ -1223,8 +1230,9 @@ cdef inline tr_datetime64_to_ft(flexible_type& ret, v):
     # Since flexible type datetime only goes down to microseconds, convert to
     # this. If higher resolution, this will truncate values
     cdef object as_py_datetime = v.astype('M8[us]').astype('O')
-    as_py_datetime = as_py_datetime.replace(tzinfo=GMT(0))
-    tr_datetime_to_ft(ret, as_py_datetime)
+    if as_py_datetime is not None:
+        as_py_datetime = as_py_datetime.replace(tzinfo=GMT(0))
+        tr_datetime_to_ft(ret, as_py_datetime)
```

Comment:
What happens if as_py_datetime is None?  We should at least clear ret to make it clear what happens.